provider/resources.go

```diff
@@ -183,6 +183,11 @@ func Provider() tfbridge.ProviderInfo {
 
 	prov.SetAutonaming(255, "-")
 
+	prov.SkipExamples = func(args tfbridge.SkipExamplesArgs) bool {
+		// TODO[pulumi/pulumi#13108]: work around PANIC printed in an example
+		return args.Token == "hcloud:index/getSshKeys:getSshKeys"
+	}
```

Comment:
https://github.com/pulumi/pulumi/issues/13108 has been fixed. I'm afraid that this will have us skipping the example for all time. Instead of skipping the example, can we upgrade the required version of github.com/pulumi/pulumi/pkg/v3 to master?

Reply:
We cannot do that presently as it fails bridge compilation as there's some breaking changes to GetMapping call. The bridge has to upgrade first, and basically pulumi/pkg has to release first.